pkg/cmd/initialize/sigs.go

```diff
@@ -70,6 +79,7 @@ func CreateEventsFromSignatures(startId events.ID, sigs []detect.Signature) {
 				events.Capabilities{},
```

Comment:
Yes, it is,https://github.com/aquasecurity/tracee/blob/main/pkg/cmd/initialize/sigs.go#L64C1-L65C1 and I think it is correct considering its purpose on tracee, but for signature it might be different, what info you were thinking should be here?

Reply:
AH, nice! https://github.com/aquasecurity/tracee/blob/main/types/detect/detect.go#L39C1-L40C1 

Will change it